sqrl-planner/sqrl-planner-local/src/main/java/com/datasqrl/frontend/SqrlPlan.java

```diff
@@ -90,7 +92,8 @@ public Namespace plan(ScriptNode node, List<ModuleLoader> additionalModules,
         throw e;
       } catch (Exception e) {
         //Print stack trace for unknown exceptions
-        if (e.getMessage() == null) {
+        if (e.getMessage() == null || e instanceof IllegalStateException
```

Comment:
Let's do this at the top. I added this to AbstractCommand and added a similar method called "isInternalError" to CollectedException. That's a more general way to handle this instead of doing it in multiple places in the codebase.